### DISCUSSION TITLE
18.12.23

### DISCUSSION THREAD

**Discussion by Roadraudi** [Roadraudi/Codespace_Repository#9] [Announcements]
In der heutigen Vorlesung haben wir uns zuerst mit For Loops beschäftigt und dabei den Befehl range() genauer unter die Lupe genommen. Um dies auch zu üben habe ich ein kleines Programm programmiert das einen Tannenbaum sowie ein Dreieck aus X generiert. Den Code findet ihr im Source Ordner.
Zudem haben wir uns noch mit Turtle in Python auseinandergesetzt und gewisse Beispiel Programme ausgeführt sowie gewisse Parameter verändert um zu schauen was  diese bewirken.

Bisher kam als Code der BMI-Rechner und ein Zeit-Umwandler dazu. Als Ausblick für die kommende Zeit würde ich gerne noch ein oder zwei Sortieralgorithmen programmieren wie der Bubblesort oder Quicksort. Außerdem wie in dem anderen Announcement schon angekündigt wurde jetzt endlich Git erfolgreich verbunden und ich werde mich jetzt an größere Projekte dran wagen und dann stückweise Code hochladen. Als Idee stehen kleinere und relativ einfache Spiele im Raum wie Schere, Stein und Papier oder sowas...

Also bleibt spannend die nächste Zeit!

**Comment by reco74** [comment]
Ich finde es spannend was du so gemacht hast im Unterricht und finde es cool, dass du deine Ergebnisse hochgeladen hast!
Deine Codes hören sich auch spannend und interessant an. Vielleicht werde ich da mal reinschauen.

Viel Spaß beim Coden!

**Comment by JonNoe** [comment]
Schön, dass dein Git endlich mit deiner Programmiersoftware verbunden ist, jetzt musst du es nicht mehr manuell hochladen, das ist echt praktisch. Deine bisherigen Ideen von Programmen klingen spannend und ich finde es gut wie übersichtlich deine Codes (bspw. der BMI Rechner ist) .

Freue mich schon schon auf deine nächsten Programme
